7.3.частые_сценарии/7.3.7.py

```diff
@@ -0,0 +1,6 @@
+from math import *
```

Comment:
Лучше импортировать только нужные нам функции:
```suggestion
from math import log
```